.github/PULL_REQUEST_TEMPLATE/devconsole_pr.md

```diff
@@ -0,0 +1,24 @@
+### [User Story/Defect/Task]: add JIRA link
+<!-- For e.g [User Story]: https://jira.coreos.com/browse/ODC-100 -->
```

Comment:
This should be simplified to a single line that the dev doesn't have to delete the title. Every other title is valid to remain but we don't want PR descriptions containing text `add JIRA link`

`Fixes: <Insert Jira link eg. https://issues.redhat.com/browse/ODC-XXX>`

Reply:
Removed all h3 & aligned everything to this format. Also fixed task list issue.